lsstswBuild.sh

```diff
@@ -7,12 +7,15 @@
 #  expectations regarding the 'work' directory location are  equivalent.
 # /\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\
 
+set -e
+
 SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
```

Comment:
I guess this is a cheap way of resolving symlinks without having to do the cross-OS crap you need to get realpath to work?